web/impact/impact/tests/test_graphql.py

```diff
@@ -154,7 +153,7 @@ def test_mentor_program_role_grants_only_returns_mentor_roles(self):
             program_role__user_role=_mentor, person=user)
         program = mentor_role_grant.program_role.program
         program_overview_link = program.program_overview_link
-        judge_role_grant = ProgramRoleGrantFactory.create(
+        ProgramRoleGrantFactory.create(
```

Comment:
Good catch, I think that assignment was bit of cruft I neglected to clean up in AC-7771.